Content.Server/Shuttles/Components/ThrusterComponent.cs

```diff
@@ -50,11 +50,17 @@ public sealed partial class ThrusterComponent : Component
 
         public bool Firing = false;
 
+        /// <summary>
+        /// How often thruster deals damage.
+        /// </summary>
+        [DataField]
+        public TimeSpan FireCooldown = TimeSpan.FromSeconds(2);
+
         /// <summary>
         /// Next time we tick damage for anyone colliding.
         /// </summary>
-        [ViewVariables(VVAccess.ReadWrite), DataField("nextFire", customTypeSerializer:typeof(TimeOffsetSerializer))]
-        public TimeSpan NextFire;
+        [DataField(customTypeSerializer: typeof(TimeOffsetSerializer))]
```

Comment:
```suggestion
        [DataField(customTypeSerializer: typeof(TimeOffsetSerializer)), AutoPausedField]
```
needs `AutoPausedField`
and the component needs `AutoGenerateComponentPause`